### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by edenlightning**

### CODE REVIEW

File: src/lightning_fabric/README.md [hunk 1]
```diff
@@ -1 +1,167 @@
...
+
+```bash
+pip install -U lightning
+```
+
+## Convert your PyTorch to Fabric
+
+1. Create the `Fabric` object at the beginning of your training code.
+
+```
```
Comment (awaelchli) [inline src/lightning_fabric/README.md:92]: Nice!
If you indent the code blocks by 4 spaces, aligning it with the start of the bullet point "Create the Fabric...", the code blocks will be better aligned with the bullet points in the rendered output :)

File: src/lightning_fabric/README.md [hunk 2]
```diff
@@ -1 +1,167 @@
...
+
+# FAQ
+
+## When to use Fabric?
+
+- **Minimum code changes**- You want to scale your PyTorch model to use multi-GPU or use advanced strategies like DeepSpeed without having to refactor. You don’t care about structuring your code- you just want to scale it as fast as possible.
+- **Maxmium control**- Write your own training and/or inference logic down to the individual optimizer calls. You aren’t forced to conform to a standardized epoch-based training loop like the one in Lightning Trainer. You can do flexible iteration based training, meta-learning, cross-validation and other types of optimization algorithms without digging into framework internals. This also makes it super easy to adopt Fabric in existing PyTorch projects to speed-up and scale your models without the compromise on large refactors. Just remember: With great power comes a great responsibility.
+- **Maxmium flexibility**- You want to have full control over your entire training- in Fabric all features are opt-in, and it provides you with a tool box of primitives so you can build your own Trainer.
+
+## When to use the [Lightning Trainer](https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html)?
```
Comment (awaelchli) [inline src/lightning_fabric/README.md:137]: Let's make sure we replace all links pointing to https://pytorch-lightning.readthedocs.io/en/stable with https://lightning.ai/docs/pytorch/stable and https://lightning.ai/docs/fabric/stable :))

**Review by awaelchli — COMMENTED** [review]
🎉

File: src/lightning_fabric/README.md [hunk 3]
```diff
@@ -1 +1,167 @@
...
+
+______________________________________________________________________
+
+# Examples
+
+- GAN
+- Meta learning
+- Large language models
+- Reinforcement learning
+- Active learning
```
Comment (Borda) [inline src/lightning_fabric/README.md:157]: Shall these be links?

**Review by awaelchli — APPROVED** [review]

**Comment by awaelchli** [comment]
Resolved merge conflict and fixed a small alignment issue! 
Looks great! 2.0 here we come

**Review by Borda — APPROVED** [review]

**Pull request merged**